packages/manager/apps/dedicated/client/app/ip/byoip/byoip.routing.js

```diff
@@ -4,7 +4,7 @@ export default /* @ngInject */ ($stateProvider) => {
   $stateProvider.state('app.ip.byoip', {
     url: '/byoip',
     views: {
-      byoip: ipByoipComponent.name,
+      '': ipByoipComponent.name,
```

Comment:
what does this empty string mean here, can't we use some good name?

Reply:
I have a common ui-view for both byoip component and iplist page. hence using blank in both location without providing any name.